### PR TITLE
Add The DevPool 

### PR DESCRIPTION
**Pull request by Draeieg**

### CODE REVIEW

File: src/resource.json [hunk 1]
```diff
@@ -2769,5 +2769,16 @@
     "tag_en": "DeFi",
     "logo": "/images/defi.png",
     "image": ""
-  }
+  },
+  {
+    "name": "Ubiquity DevPool",
+    "name_en": "Ubiquity DevPool",
+    "desc": "Solve bounties, find work and blitzscale you project with our solution to efficiently run natively decentralized organizations",
```
Comment (pavlovcik) [inline src/resource.json:2776]:
```suggestion
    "desc": "Solve bounties, find work, and blitzscale your project to efficiently run natively decentralized organizations.",
```

File: src/resource.json [hunk 2]
```diff
@@ -2769,5 +2769,16 @@
...
+    "desc_en": "Solve bounties, find work and blitzscale you project with our solution to efficiently run natively decentralized organizations",
```
Comment (pavlovcik) [inline src/resource.json:2777]:
```suggestion
    "desc_en": "Solve bounties, find work, and blitzscale your project to efficiently run natively decentralized organizations.",
```

File: src/resource.json [hunk 3]
```diff
@@ -2769,5 +2769,16 @@
...
+    "url": "https://dao.ubq.fi/devpool",
+    "tag": "Tools & Resources",
+    "tag_en": "Tools & Resources",
+    "logo": "/images/Ubq.png",
```
Comment (pavlovcik) [inline src/resource.json:2781]: I think you need to add an image to this pull request.

File: src/resource.json [hunk 4]
```diff
@@ -2769,5 +2769,16 @@
...
+    "image": ""
+  {
 ]
```
Comment (pavlovcik) [inline src/resource.json:2784]: I think you broke the json object. Careful with your syntax.

**Review by pavlovcik — CHANGES_REQUESTED** [review]

File: src/resource.json [hunk 5]
```diff
@@ -2769,5 +2769,16 @@
...
+    "name": "Ubiquity DevPool",
+    "name_en": "Ubiquity DevPool",
+    "desc": "Solve bounties, find work and blitzscale you project with our solution to efficiently run natively decentralized organizations",
+    "desc_en": "Solve bounties, find work and blitzscale you project with our solution to efficiently run natively decentralized organizations",
+    "url": "https://dao.ubq.fi/devpool",
+    "tag": "Tools & Resources",
+    "tag_en": "Tools & Resources",
+    "logo": "/images/Ubq.png",
+    "image": ""
+  {
```
Comment (pavlovcik) [inline src/resource.json:2783]:
```suggestion
  }
```

**Comment by pavlovcik** [comment]
@poria-cat I implemented the necessary changes.

**Comment by poria-cat** [comment]
@pavlovcik Cool, wishing DevPool all the best!

**Comment by poria-cat** [comment]
@pavlovcik New json already deployed, just find it in Tools & Resources.
When we upgrade to v2, the items will be arranged in a more logical way.

**Comment by pavlovcik** [comment]
How can we get listed under partnership? The location of our information is hidden and difficult to find.

**Comment by poria-cat** [comment]
@pavlovcik partnership project had a collaboration with xdai a long time ago and it's hard to adjust them. 
I just adjusted the DevPool position in Tools so that no extra clicks are needed.

**Pull request merged**